components/google_drive/actions/create-file/create-file.js

```diff
@@ -0,0 +1,248 @@
+const googleDrive = require("../../google_drive.app");
+const fs = require("fs");
+const got = require("got");
+
+module.exports = {
+  key: "google_drive-create-file",
+  name: "Create a New File",
+  description: "Create a new file from a URL or /tmp/filepath.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    googleDrive,
+    drive: {
+      propDefinition: [
+        googleDrive,
+        "watchedDrive",
+      ],
+    },
+    parents: {
+      type: "string",
+      label: "Parent Folder",
+      description:
+        "The ID of the parent folder which contain the file. If not specified as part of a create request, the file will be placed directly in the user's My Drive folder.",
```

Comment:
Typo (`contain` should be `contains`). Also wrap this and any other long lines to 100 chars.

Reply:
Updated